src/urllib3/request.py

```diff
@@ -91,23 +103,23 @@ def request_encode_url(
         if headers is None:
             headers = self.headers
 
-        extra_kw = {"headers": headers}
+        extra_kw: Dict[str, Any] = {"headers": headers}
         extra_kw.update(urlopen_kw)
 
         if fields:
-            url += "?" + urlencode(fields)
+            url += "?" + urlencode(fields)  # type: ignore
```

Comment:
`error: Argument 1 to "urlencode" has incompatible type "Union[Sequence[Union[Tuple[str, Union[Union[str, bytes], Tuple[str, Union[str, bytes]], Tuple[str, Union[str, bytes], str]]], RequestField]], Dict[str, Union[Union[str, bytes], Tuple[str, Union[str, bytes]], Tuple[str, Union[str, bytes], str]]]]"; expected "Union[Mapping[Any, Any], Mapping[Any, Sequence[Any]], Sequence[Tuple[Any, Any]], Sequence[Tuple[Any, Sequence[Any]]]]"`